pyserver/monitor_client.py

```diff
@@ -0,0 +1,212 @@
+from flask import Flask, render_template, request, jsonify
+import socket
+import random
+import json
+import sys
+import functools 
+import operator
+from ctypes import *
+import struct
+import time
+
+""" This class defines a C-like struct """
+class IMSI_REQ(Structure):
+    _fields_ = [("req_type", c_int),
+                ("imsi", c_byte*16)]
+
+class IMSI_STRUCT(Structure):
+    _fields_ = [("imsi", c_byte*16)]
+
+class PLMN(Structure):
+    _fields_ = [("idx", c_byte*3)]
+
+class TAI(Structure):
+    _fields_ = [("plmn_id", PLMN),
+                ("tac", c_byte*2)]
+
+class AMBR(Structure):
+    _fields_ = [("max_req_dl", c_int),
+                ("max_req_ul", c_int)]
+
+class IMSI_RSP(Structure):
+    _fields_ = [("result", c_int), 
+                ("imsi", c_byte*16),
+                ("bearer_id", c_char),
+                ("tai", TAI),
+                ("ambr", AMBR)]
+
+class MyEncoder(json.JSONEncoder):
+    def default(self, o):
+        if type(o) is bytes:
+            return o.decode("ISO-8859-1")
+        return super(MyEncoder, self).default(o)
+
+app = Flask(__name__)
+
+@app.route("/")
+
+def helloWorld():
```

Comment:
may be we should get rid of this or put some other banner ?

Reply:
done